.travis.yml

```diff
@@ -1,24 +1,29 @@
 language: php
- 
+
 php:
-  - 5.3
   - 5.4
   - 5.5
   - 5.6
-  - 7
+  - 7.0
+  - 7.1
+  - 7.2
+  - nightly
   - hhvm
   - hhvm-nightly
 
 matrix:
   fast_finish: true
+  include:
+    - php: 5.3
+      dist: precise
   allow_failures:
-    - php: 7
+    - php: nightly
     - php: hhvm
     - php: hhvm-nightly
 
 before_script:
   - composer self-update
-  - composer install --dev --prefer-source --no-interaction
+  - composer update --prefer-source --no-interaction
```

Comment:
Why update and not install?

Reply:
Because of the `composer.lock` is existed, I also need to let the locked dependencies should be updated in every Travis CI build.